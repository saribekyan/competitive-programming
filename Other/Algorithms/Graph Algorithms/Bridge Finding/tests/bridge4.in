50 100
14 7
25 7
30 43
31 1
42 2
28 1
23 18
20 27
10 37
13 38
28 15
11 44
40 41
43 31
33 9
32 2
6 20
34 39
46 24
30 35
16 2
49 18
19 8
15 17
40 15
34 15
10 13
28 39
37 2
40 48
34 18
30 26
20 25
49 2
45 29
15 5
5 32
13 41
20 19
2 14
36 17
26 28
18 25
18 17
28 32
18 37
43 1
26 23
39 9
14 3
26 12
38 20
43 28
34 14
17 4
35 2
37 7
20 39
13 49
21 42
36 3
24 8
39 18
15 42
45 25
29 24
47 27
11 9
10 1
48 36
19 5
20 50
5 3
46 17
38 43
47 48
9 23
3 50
6 26
27 32
34 22
15 35
15 6
43 33
3 12
40 11
12 36
24 49
7 50
25 21
44 33
48 11
31 9
38 36
39 23
47 18
40 27
45 34
26 29
10 4
