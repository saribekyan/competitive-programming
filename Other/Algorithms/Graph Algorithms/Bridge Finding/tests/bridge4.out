2 16
22 34
